src/main/java/de/tum/in/www1/artemis/repository/ExerciseGroupRepository.java

```diff
@@ -16,7 +16,8 @@
 @Repository
 public interface ExerciseGroupRepository extends JpaRepository<ExerciseGroup, Long> {
 
-    List<ExerciseGroup> findByExamId(Long examId);
+    @Query("SELECT e FROM ExerciseGroup e WHERE e.exam.id = :#{#examId}")
+    List<ExerciseGroup> findByExamId(@Param("examId") Long examId);
```

Comment:
You could then also remove `@Param("examId")` here

Reply:
done